src/index.js

```diff
@@ -78,8 +78,16 @@ class Store {
         handler(payload)
       })
     })
-    if (!options || !options.silent) {
-      this._subscribers.forEach(sub => sub(mutation, this.state))
+    this._subscribers.forEach(sub => sub(mutation, this.state))
+
+    if (
+      process.env.NODE_ENV !== 'production' &&
```

Comment:
As vuex currently does not resolve `NODE_ENV` check on the build time, we should not introduce it?
This line would cause runtime error if we use Vuex from `<script>` tag.
Or can we update build configuration for it?